pages/common/git-authors.md

```diff
@@ -0,0 +1,18 @@
+# git authors
+
+> Display log or list of committers of a Git repository. Sorted by amount of commits.
+> Part of `git-extras`.
+> Creates an AUTHORS file in the Git repository, this will be used for the command.
```

Comment:
Unfortunately, we can't have more thant 3 total description lines (including the link) :/
```suggestion
> Display log or list of committers of a Git repository. Sorted by amount of commits.
> Part of `git-extras`.
```
The `AUTHORS` line isn't even true for all possible invocations of the command (running it with `--list` doesn't operate on `AUTHORS`, I don't think)

See https://github.com/tldr-pages/tldr/pull/5807#discussion_r617511072 for details.

Reply:
You're right, the list doesn't use a file

Reply:
take a look at the commit after this